Makefile

```diff
@@ -54,10 +59,16 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 	BUNDLE_GEN_FLAGS += --use-image-digests
 endif
 
+# Image registry url to use
+REGISTRY_URL ?= quay.io
+# Image registry org to use
+REGISTRY_ORG ?= maistra-dev
 # Image hub to use
-HUB ?= quay.io/maistra-dev
```

Comment:
why split this up? it's a bit confusing, because if my env defines `HUB=something`, that will not be picked up below because there it uses `REGISTRY_URL` and `REGISTRY_ORG`

Reply:
True, I'm going to split the `HUB` in two vars afterwards